en/guides/developers/tested-configure-new-programming-language/README.md

```diff
@@ -0,0 +1,1070 @@
+---
+title: "[nl] Configureren nieuwe programmeertaal TESTed"
+description: "[nl] Configureren nieuwe programmeertaal TESTed"
+---
+
+::: warning Sorry
+For now, this page is only available in Dutch. Sorry!
+:::
+
+# Configuratie van een programmeertaal
+In deze handleiding wordt in detail uitgelegd hoe een nieuwe programmeertaal aan TESTed kan toegevoegd worden.
+We doen dat door te beschrijven hoe de programmeertaal C toegevoegd is.
+Enkele nuttige links die hierbij kunnen helpen zijn:
+
+- Bestaande configuraties: <https://github.com/dodona-edu/universal-judge/tree/master/tested/languages>
+- Testoefeningen: <https://github.com/dodona-edu/universal-judge/tree/master/exercise>
+
+In deze referentie worden regelmatig codebestanden getoond, verspreid over meerdere codefragmenten
+(bijvoorbeeld eerst `n` regels, daarna wat tekst en dan pas de rest van de regels).
+
+## TESTed lokaal uitvoeren
+Tijdens het configureren van een programmeertaal is het nuttig om TESTed lokaal uit te voeren,
+zonder daarvoor het volledige Dodona-platform te moeten uitvoeren.
+Buiten de _dependencies_ voor de bestaande programmeertalen is TESTed een Python-package,
+dat op de normale manier uitgevoerd kan worden.
```

Comment:
I think we want assume to assume that, because when adding a programming language TESTed, we expects some knowledge about Python.
This information is also found in the README of the TESTed repo.